app/mailers/rrd_completed_mailer.rb

```diff
@@ -4,7 +4,6 @@ class RrdCompletedMailer < ApplicationMailer
   def build(submission)
     @submission = submission
     @disability_struct = RapidReadyForDecision::Constants.first_disability(submission) || {}
-    @disability_code = @disability_struct['code']
```

Comment:
Root cause of error: should have used symbol like so: `@disability_struct[:code]`